templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,31 @@
+# Configure authentication methods
+
+You can configure your organization settings to allow users to log in using
+different methods. Users may already have accounts with some of the
+authentication methods and others can be integrated with existing systems.
```

Comment:
I don't think the wording is exactly right here, as it might confuse a user who's unfamiliar with authentication protocols.

You can authenticate yourself using different authentication protocols (e.g. LDAP); in the case of OAuth2, that framework enables applications such as Zulip to integrate with HTTP services like GitHub and Google.

I suggest that you read up on authentication protocols a bit more and before editing your introduction. Good effort though!